src/main.rs

```diff
@@ -116,6 +131,13 @@ pub struct SendArgs {
     /// being shared.
     pub path: PathBuf,
 
+    /// Whether to use a short ticket that doesn't contain information
+    /// about your IP addresses.
+    ///
+    /// Useful if you want smaller tickets or want to test node discovery mechanisms.
+    #[clap(long, action = clap::ArgAction::SetTrue)]
+    pub short_ticket: bool,
```

Comment:
Can we just use https://docs.rs/iroh/latest/iroh/base/node_addr/enum.AddrInfoOptions.html here? It seems to support FromStr, so it should just work here.